tree.go

```diff
@@ -495,7 +495,8 @@ func (n *InternalNode) ComputeCommitment() *bls.G1Point {
 	}
 
 	var commP *bls.G1Point
-	if n.treeConfig.nodeWidth-emptyChildren >= multiExpThreshold {
+	if (n.treeConfig.width == 10 && n.treeConfig.nodeWidth-emptyChildren >= multiExpThreshold10) ||
```

Comment:
I think it would make sense to set `multiExpThreshold` as a member of `TreeConfig`. What do you think?

Reply:
Good idea, moved it.